DigitalLearningSolutions.Web/Views/Login/ChooseACentre.cshtml

```diff
@@ -15,65 +15,80 @@
 
     <h1 id="page-heading" class="nhsuk-heading-xl">Choose a centre</h1>
     <p class="nhsuk-body-l">
-      You are registered at more than one centre. Please choose the centre you would like to sign in to.
+      Please choose the centre you would like to sign in to.
     </p>

```

Comment:
Do we still want to display this when there are no centres to choose from? (Either 0 accounts, or they have accounts but none of them have a "Choose" button)?

Reply:
I have moved both that and the inset-text inside the `if (Model.CentreUserDetails.Count > 0)`